### ISSUE TITLE
Metroid Prime Remastered Water Texture Not Fully Loading

### ISSUE THREAD

**Issue by jon-mil-92** [yuzu-emu/yuzu#9797]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Affected Build(s)

Early Access All Recent Builds

### Description of Issue

In Phendrana Drifts area, the water texture will not fully load. The texture is striped across the water surface.

### Expected Behavior

Yuzu:
<img width="1281" alt="Bad" src="https://user-images.githubusercontent.com/111253795/218525560-6963d5df-31b8-4e98-880c-1f8a1bb2ebf3.PNG">

Ryujinx:
<img width="1038" alt="Good" src="https://user-images.githubusercontent.com/111253795/218525601-70bf2870-f7ef-46ff-aa44-54d5a0a150cc.PNG">

### Reproduction Steps

No reproduction steps. It just occurs.

### Log File

[yuzu_log.txt](https://github.com/yuzu-emu/yuzu/files/10724595/yuzu_log.txt)

### System Configuration

CPU: 5800X3D
GPU/Driver: 6900XT 22.11.2
RAM: 32 GB G.Skill TridentZ DDR4 4400MHz CL17
OS: Windows 10

**Comment by FernandoS27** [comment]
Could you provide a save and instructions on how to get there?

**Comment by jon-mil-92** [comment]
> Could you provide a save and instructions on how to get there?

Yes. I have attached a save file. To get to the area, just open the door and head straight out to the open area.
[save.zip](https://github.com/yuzu-emu/yuzu/files/10726480/save.zip)

**Comment by TriVoxel** [comment]
Here is a video I captured that shows some very odd behavior of this water: https://youtu.be/NxK18LfFLQg

Hopefully this helps.

**Comment by Rdp157** [comment]
I can confirm, its related to upscaling, at 2x all textures disappear, at 3x it shows as only strips, but i was able to safely render the water at 1.5x.

**Comment by jon-mil-92** [comment]
> I can confirm, its related to upscaling, at 2x all textures disappear, at 3x it shows as only strips, but i was able to safely render the water at 1.5x.
> 
> edit: NVM, after a while they´re gone again, it seens we must run at 1x in pendrana

Thanks! I updated the reproduction steps in the original post to reflect your findings.

**Comment by Rdp157** [comment]
> > I can confirm, its related to upscaling, at 2x all textures disappear, at 3x it shows as only strips, but i was able to safely render the water at 1.5x.
> > edit: NVM, after a while they´re gone again, it seens we must run at 1x in pendrana
> 
> Thanks! I updated the reproduction steps in the original post to reflect your findings.

![Sem título](https://user-images.githubusercontent.com/125580242/219419853-f6d7aa35-5686-4f4a-8442-2cc069d86256.png)

here´s a picture to prove, EA 3404, 1x IR, Vulkan, SMAA, FSR,  GPU NORMAL 16x AF

**Comment by Boodax78** [comment]
There is another place in the game where a similar bug occurs when you increase the scale after x1.

The water in the Life Grove in the Tallon Overworld.

**Comment by jon-mil-92** [comment]
I noticed this as well

**Comment by JonnieCache** [comment]
Also seeing this, at 3x theyre narrow strips.

**Comment by TriVoxel** [comment]
> There is another place in the game where a similar bug occurs when you increase the scale after x1.
> 
> The water in the Life Grove in the Tallon Overworld.

This water actually renders fine for me at 2x. It was only when I changed the resolution while the game was running that it disappeared IIRC.

**Comment by TriVoxel** [comment]
https://user-images.githubusercontent.com/25536957/219908170-3c056c41-e63d-4c8e-bdd2-760e27cbec27.mp4

**Comment by Graslu** [comment]
I've only noticed three rooms with broken water during my playthrough at 3x scaling:

- Phendrana Shorelines (Phendrana)
- Hunter Cave Access (Phendrana)
- Life Grove (Tallon Overworld)

Every other room with water seems to display it correctly.
![image](https://user-images.githubusercontent.com/34986384/220229499-1ec4009f-baa3-4814-a416-22fd6fdafc97.png)
![Desktop Screenshot 2023 02 21 - 03 03 54 61](https://user-images.githubusercontent.com/34986384/220229524-5413e5a5-73a8-4672-b07c-51c257ceddbf.png)
![Desktop Screenshot 2023 02 21 - 03 06 29 18](https://user-images.githubusercontent.com/34986384/220229534-3ec40746-d439-4d6a-af12-47023e29ace8.png)

**Comment by jon-mil-92** [comment]
I think I saw a tiny patch of water somewhere in the chozo ruins that has
the issue as well.

**Comment by joebellagio79** [comment]
Yes I can reproduce this at 3x in the Sun Tower and the Tower of Light
![MetroidPrime_Glitch_In_SunTowerAccess_Ryujinx1_1_644](https://user-images.githubusercontent.com/126494501/221606176-882bf876-e35d-4732-ab3c-b5fbcc4d1a8c.png)
![MetroidPrime_Glitch_In_TowerOfLight_Ryujinx1_1_644](https://user-images.githubusercontent.com/126494501/221606203-227f9eea-8682-46d4-bd51-01c57266b7ef.png)

**Comment by Graslu** [comment]
That's strange, I cannot reproduce that. All bodies of water except the ones I listed earlier render just fine for me since the day of this report.

![image](https://user-images.githubusercontent.com/34986384/221625968-6f3470e1-1fbe-4fe7-b2b4-dda26461127b.png)


![image](https://user-images.githubusercontent.com/34986384/221625300-8ea5ed79-1e2d-4133-b3c5-0bcf4fcccb5a.png)

**Comment by joebellagio79** [comment]
Yeah it's weird how if affects different people in different areas. Mine seems fine in other bodies of water, at least so far.

**Comment by Unai24** [comment]
This issue is still happening in latest builds. The 4K patch from Theboy181 is not a good solution because it makes the game stutter when loading ASTC textures like the visor effects, as it upscales those textures too. Yuzu doesn't upscale them since certain build, so the game runs flawlesly in those cases at 2X resolution.

The only acceptable solution right now is playing the game at 1X with the 1080p mod, which is a shame considering how nice it looks and runs at 2X.

**Comment by jon-mil-92** [comment]
The issue is even worse now unfortunately. Decided to pick the game back up recently. On the latest early access build as of today, no water textures render. I have tested 1x, 2x, 3x, and "theboy" 1080p and 4k mods. No water textures on any of them. I don't have time to test anymore configurations at the moment, but if I do, I'll post the results.

**Comment by spyro2000** [comment]
Same here on 1673, Vulkan, 3x
